src/controllers/PostController.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from "express";
+import { IPostRepository } from "../repositories/ContentfulRepository.ts";
+
+class PostController {
+  constructor(postRepository: IPostRepository) {
+    this.postRepository = postRepository;
```

Comment:
If I'm not mistaken, you can do something like `private _postRepository: IPostRepository` as constructor parameters and you won't need boilerplate assignment